clients/apps/web/src/components/Products/CreateProductModal.tsx

```diff
@@ -50,8 +49,7 @@ export const CreateProductModal = ({
 
   const [enabledBenefitIds, setEnabledBenefitIds] = useState<
     BenefitPublicInner['id'][]
-    // Pre-select premium articles benefit
-  >(organizationBenefits.filter(isPremiumArticlesBenefit).map(({ id }) => id))
+  >([])
```

Comment:
Removed the pre-selected premium benefit here, as it doesn't make sense to pre-populate when creating new products.